packages/trip-form/src/MetroModeSelector/AdvancedModeSettingsButton/index.tsx

```diff
@@ -108,17 +108,30 @@ const AdvancedModeSettingsButton = ({
   const intl = useIntl();
   const label = generateModeButtonLabel(modeButton.key, intl, modeButton.label);
   const checkboxId = `metro-submode-selector-mode-${id}`;
+  const containsSubsettings = modeButton.modeSettings.length > 0;
+
+  const subsettingsStatusLabel = modeButton.enabled
+    ? intl.formatMessage({
+        id: "otpUi.ModeSelector.labels.subsettingsExpanded"
+      })
+    : intl.formatMessage({
+        id: "otpUi.ModeSelector.labels.subsettingsCollapsed"
+      });
+  const ariaLabel = containsSubsettings
+    ? label + subsettingsStatusLabel
+    : label;
+
   return (
     <SettingsContainer className="advanced-submode-container">
       <StyledModeSettingsButton
         accentColor={accentColor}
         className="advanced-submode-mode-button"
         fillModeIcons={fillModeIcons}
         id={modeButton.key}
-        subsettings={modeButton.modeSettings.length > 0}
+        subsettings={containsSubsettings}
       >
         <input
-          aria-label={label}
+          aria-label={ariaLabel}
```

Comment:
Improvement: We probably don't need `aria-label`, instead you could place the `subsettingsStatusLabel` inside an `{" "}<InvisibleA11yLabel>` within the `<label>` element below. That way, you can remove the leading space from the i18n content.

Reply:
Because `<InvisibleA11yLabel>` would be a [nested element inside a label](https://stackoverflow.com/questions/72090573/safari-voiceover-not-reading-labels-with-nested-elements-correctly), screenreaders read it as "Transit and one other item" instead of the full string. This was the easiest way I could find to make sure the full string and no other confusing information was presented.

Reply:
Good point, could you add a comment about that, please?

Reply:
Added in [61be29d](https://github.com/opentripplanner/otp-ui/pull/852/commits/61be29d78b927eca4dbde0a5b350ddd442afae36)